app/models/allocated_offender.rb

```diff
@@ -25,6 +25,8 @@ class AllocatedOffender
            to: :@allocation
   delegate :full_name_ordered, to: :staff_member, prefix: true
 
+  attr_reader :offender
```

Comment:
I think this is already declared in the SortableAllocation module. Shouldn't cause any issues tho even if it is dupe.